.github/workflows/ci.yml

```diff
@@ -37,19 +37,28 @@ jobs:
       - name: Lint
         run: npm run lint
 
-      - name: Test
+      - name: Tests & Coverage
         if: github.ref != 'refs/heads/master'
         run: npm run test
-
-      - name: Test & Coverage
-        if: github.ref == 'refs/heads/master'
-        run: |
-          npm run test
-          export COVERALLS_REPO_TOKEN=${{ secrets.COVERALLS_REPO_TOKEN }}
-          npm run coverage
+        run: npm run coverage
```

Comment:
I think this was not intended

you need to write 

```yml
      - name: Tests & Coverage
        if: github.ref != 'refs/heads/master'
        run: |
          npm run test
          npm run coverage
```

Reply:
You are right bro